scripts/hail_batch/ld_prune/hgdp_1kg_ld_prune.py

```diff
@@ -22,6 +22,9 @@ def query(output):
     # reproduce gnomAD genotype filtering
     mt = annotate_adj(mt)
     mt = mt.filter_entries(mt.adj)
+    # Filter to common variants
+    mt = hl.variant_qc(mt)
```

Comment:
Doesn't this table already contain allele-frequency information?

Reply:
I don't think so. Which row field are you referring to?

Reply:
There's a `cohort_freq.AF` and `gnomad_freq.AF` field, as well as `gnomad_popmax.AF`.

Reply:
that's for the entire cohort. The values aren't individual allele frequencies for each variant